cubano-concordion/src/test/resources/demo/HelloWorld.md

```diff
@@ -1,4 +1,6 @@
 # Hello World
 
-## [Example Failure](-)
-[Hello World!](- "?=getGreetingFailure()")
+## [Example 1](-)
+[Hello World!](- "?=getGreeting()")
```

Comment:
This isn't a good Spec example. If you want to keep as "hello world" can you change the fixture to search for hello world, and make it more descriptive? The Spec doesn't match the high standards that are espoused in the documentation :)

Reply:
Spec now matches fixture.  This is more for me to play with so I can test changes manually but good to be consistent